openlibrary/plugins/openlibrary/js/bulk-tagger/BulkTagger.js

```diff
@@ -621,4 +622,32 @@ export class BulkTagger {
         this.tagsToAdd = []
         this.tagsToRemove = []
     }
+
+    highlightMenuOption(menuOption) {
+        const menuOptionElement = menuOption.rootElement;
+
+        switch (menuOption.optionState) {
+        case MenuOptionState.NONE_TAGGED:
+            menuOptionElement.classList.remove(
+                'highlight-some-tagged',
+                'highlight-all-tagged'
+            );
+            menuOptionElement.classList.add('highlight-none-tagged');
+            break;
+        case MenuOptionState.SOME_TAGGED:
+            menuOptionElement.classList.remove(
+                'highlight-none-tagged',
+                'highlight-all-tagged'
+            );
+            menuOptionElement.classList.add('highlight-some-tagged');
+            break;
+        case MenuOptionState.ALL_TAGGED:
+            menuOptionElement.classList.remove(
+                'highlight-none-tagged',
+                'highlight-some-tagged'
+            );
+            menuOptionElement.classList.add('highlight-all-tagged');
+            break;
+        }
+    }
```

Comment:
```suggestion
```
This much state management is not needed.  Whenever an option is clicked, add the `selected-tag--staged` class.